packages/module/src/Chatbot/Chatbot.scss

```diff
@@ -2,31 +2,13 @@
 // Chatbot
 // ============================================================================
 .pf-chatbot {
-  --pf-t--chatbot--heading--font-family: var(
-    --pf-v6-c-content--heading--FontFamily,
-    redhatdisplayvf,
-    redhatdisplay,
-    helvetica,
-    arial,
-    sans-serif
-  );
-  --pf-t--chatbot--font-size: var(--pf-t--global--font--size--md);
-  --pf-t--chatbot--background: var(--pf-t--color--gray--10);
-  --pf-t--chatbot--border: var(--pf-t--color--gray--30);
-  --pf-t--chatbot--illustration--fill: var(--pf-t--color--red--50);
-  --pf-t--chatbot--code--background: var(--pf-t--color--gray--20);
-  --pf-t--chatbot--icon--fill: var(--pf-t--color--gray--50);
-  --pf-t--chatbot--icon--fill--hover: var(--pf-t--color--gray--70);
-  --pf-t--chatbot--timing-function: cubic-bezier(0.77, 0, 0.175, 1);
-  --pf-t--chatbot--jump--background: var(--pf-t--chatbot--background);
-  --pf-t--chatbot--jump--background--hover: var(--pf-t--color--white);
-
   position: fixed;
-  bottom: 96px;
-  right: 24px;
+  inset-block-end: var(--pf-t--global--spacer--800); // no associated semantic token
+  inset-inline-end: var(--pf-t--global--spacer--lg);
+  padding: var(--pf-t--global--spacer--lg);
   display: grid;
   grid-template-rows: auto 1fr auto;
-  width: 480px;
+  width: 30rem;
```

Comment:
Note for me: We'll have to modify some of the other PRs like https://github.com/patternfly/virtual-assistant/pull/102/files that rely on this value.